File: lib/models/usuario.dart
class Usuario {
  final int id;
  String rut;
  String nombres;
  String apellidos;
  String email;
  String nickname;
  final String password;
  final DateTime fechaNacimiento;
  final String sexo;

  Usuario({
    required this.id,
    required this.rut,
    required this.nombres,
    required this.apellidos,
    required this.email,
    required this.nickname,
    required this.password,
    required this.fechaNacimiento,
    required this.sexo,
  });

}

File: backend/models/usuario.py
from sqlalchemy import Column
from sqlalchemy import Integer
from sqlalchemy import String
from sqlalchemy import Date


from sqlalchemy.orm import declarative_base


Base = declarative_base()

class Usuario(Base):
    __tablename__ = "usuarios"

    id_usuario = Column(
        Integer,
        primary_key=True)
    rut = Column(String(12))
    nombres = Column(String(100))
    apellidos = Column(String(100))
    email = Column(String(100))
    nickname = Column(String(50))
    password = Column(String(255))
    fecha_nacimiento = Column(Date)
    sexo = Column(String(20))